### PR TITLE
Remove use of ReactS

### PR DESCRIPTION
**Pull request by cquiroz**
The use of the State monad for React is no longer recommended. I've already removed a couple of uses but this PR removes the last bits

### CODE REVIEW

**Comment by cquiroz** [comment]
This indirectly remove a dependency thus we save 10k on output size

File: modules/seqexec/web/client/src/main/scala/seqexec/web/client/components/tabs/CalibrationQueueTab.scala [hunk 1]
```diff
@@ -98,10 +95,10 @@ object CalibrationQueueTab {
...
         e.preventDefaultCB *> Callback { e.dataTransfer.dropEffect = "copy" }
       },
-      ^.onDragEnter ==> b.runStateFn(onDragEnter),
-      ^.onDrop ==> b.runStateFn(onDrop),
-      ^.onDragEnd --> b.runState(onDragEnd),
-      ^.onDragLeave --> b.runState(onDragEnd),
+      ^.onDragEnter ==> onDragEnter(b) _,
+      ^.onDrop ==> onDrop(b) _,
+      ^.onDragEnd --> onDragEnd(b),
+      ^.onDragLeave --> onDragEnd(b),
```
Comment (sraaphorst) [inline modules/seqexec/web/client/src/main/scala/seqexec/web/client/components/tabs/CalibrationQueueTab.scala]: What is the difference between `==>` and `-->`?

Comment (cquiroz) [reply modules/seqexec/web/client/src/main/scala/seqexec/web/client/components/tabs/CalibrationQueueTab.scala]: In one case `-->` you only pass a `Callback` which essentially is the effect you want to execute.
`==>` is similar but you also get the `ReactEvent` which maybe useful is some cases
It is documented here
https://github.com/japgolly/scalajs-react/blob/master/doc/VDOM.md#event-handlers

Comment (sraaphorst) [reply modules/seqexec/web/client/src/main/scala/seqexec/web/client/components/tabs/CalibrationQueueTab.scala]: Thanks!

**Pull request merged**